ksp_plugin/orbit_analyser.hpp

```diff
@@ -88,9 +98,10 @@ class OrbitAnalyser {
   // Sets |analysis()| to the latest computed analysis.
   void RefreshAnalysis();
 
+  // Mutable so that the caller can call |SetRecurrence| and |ResetRecurrence|.
   Analysis* analysis();
 
-  int8_t next_analysis_percentage() const;
+  double progress_of_next_analysis() const;
```

Comment:
Comment: in 0..1.  Also, guaranteed to be == 1.0 at end of analysis.

Reply:
There is no such guarantee; commented to clarify.